docs/index.asciidoc

```diff
@@ -143,7 +143,7 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-query_template>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-result_size>> |<<number,number>>|No
 | <<plugins-{type}s-{plugin}-retry_on_failure>> |<<number,number>>|No
-| <<plugins-{type}s-{plugin}-result_on_status_>> |<<number,number list>>|No
```

Comment:
@yaauie, please validate that this fix aligns with your original intent before I merge it.